app/src/ui/changes/changed-file.tsx

```diff
@@ -12,6 +12,7 @@ interface IChangedFileProps {
   readonly oldPath?: string
   readonly include: boolean | null
   readonly availableWidth: number
+  readonly disabled: boolean
```

Comment:
This can be a bit more descriptive, maybe something along the lines of `disableSelection`